ui-library-quickstart-composites/src/App.tsx

```diff
@@ -75,8 +95,58 @@ function App(): JSX.Element {
   return <h3>Initializing...</h3>;
 }
 
-const containerStyle = {
-  height: '50%',
+const containerStyle: CSSProperties = {
+  border: 'solid 0.125rem olive',
+  margin: '0.5rem',
+  width: '50vw',
```

Comment:
Are we lenient on the use of vw and vh in the quickstart composites? @JamesBurnside

Reply:
Yeah, as long as its outside the npm package its totally fine